script.js

```diff
@@ -302,3 +306,138 @@ function searchQuestions() {
   const filteredQuestions = filterQuestions(quizQuestions, keyword);
   displayQuestionsList(filteredQuestions);
 }
+
+//entering names
+function checkInputs() {
+  const values = Array.from(playerNameInputs).map((input) =>
+    input.value.trim()
+  );
+  const isInputsFilled = values.every((value) => value !== '');
+  const isNameUnique = new Set(values).size === values.length;
+  startQuizBtn.disabled = !(isInputsFilled && isNameUnique);
+}
+
+playerNameInputs.forEach((input) =>
+  input.addEventListener('input', checkInputs)
+);
+
+//start game
+let playersData = [];
+
+function startQuiz(event) {
+  event.preventDefault();
+
+  Array.from(playerNameInputs).forEach((input) => {
```

Comment:
As you will learn in JavaScript 3, this block can be simplified using the map function.

Reply:
Good idea! I'll fix it.